src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  // static alive = []; - так виникає помилка лінтера
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive = Animal.alive || []; // тому я не використав статист. власт-ть
```

Comment:
yeah, in the current student's config lint throw an error

You might also write `Animal.alive = []` right after `Animal` class
anyway, this variant is acceptable too :+1: 

P.S. Next time do not leave such comments in the code
add them in GitHub